src/store/item-store.js

```diff
@@ -34,4 +31,8 @@ store.registerHandler('CLEAR_ALL_DATA', () => {
     store.emitChange();
 });
 
+store.registerHandler('ITEMS_RECEIVED', data => {
+    item = data.items;
```

Comment:
Shouldn't the items be stored to something like `items`, and not use `item` again?